.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
**Code Quality - Critical**
Added an .eslint file to check on my code quality and to highlight bad practices and dirty code.